nmma_api/services/retrieval_queue.py

```diff
@@ -14,6 +14,12 @@
 mongo = Mongo(**config["database"])
 retrieval_wait_time = config["wait_times"]["retrieval"]
 max_upload_failures = config["wait_times"].get("max_upload_failures", 10)
+time_limit = config["expansion"].get("time_limit", 6) * 3600  # in seconds
```

Comment:
expanse instead of expansion?

Reply:
oups :) Copilot bamboozled me. Thanks for checking!